hr_course/models/hr_course.py

```diff
@@ -180,6 +183,53 @@ def cancel_course(self):
         for record in self:
             record.write(record._cancel_course_values())
 
+    @api.model
+    def send_course_notification_email(self, partner_emails):
+        email_list = partner_emails.mapped("email")
+        company = self.env["res.company"].browse(
+            self.env.context.get("company_id") or self.env.company.id
+        )
+        channel_id = self.env.ref("hr_course.mail_channel_course_validity").id
```

Comment:
relly on the channel from company settings or use this one but then remove this one